pkg/controller/util.go

```diff
@@ -158,6 +157,8 @@ func GetVolumeCapabilities(pvSpec *v1.PersistentVolumeSpec, singleNodeMultiWrite
 		fsType := pvSpec.CSI.FSType
 		if len(fsType) == 0 {
 			fsType = defaultFSType
+			klog.Warningf("Filesystem type not found in PV spec. Will use fstype set by attacher: %v. " +
+				"This will be deprecated in future versions and defaults will be set by CSI driver.", fsType)
```

Comment:
I think we will not deprecate `--default-fstype` parameter, we will change its default to "". IMO the message should be at lower log level (it can be logged quite a lot) and just note that it's using `defaultFSType` as the fsType.